cmake/Package.cmake

```diff
@@ -0,0 +1,41 @@
+set(CPACK_PACKAGE_VENDOR "khronos")
+
+set(CPACK_PACKAGE_DESCRIPTION "OpenCL (Open Computing Language) C header files")
+
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+
+set(CPACK_RESOURCE_FILE_README "${CMAKE_CURRENT_SOURCE_DIR}/README.md")
+
+if(NOT CPACK_PACKAGING_INSTALL_PREFIX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}")
+endif()
+
+# DEB packaging configuration
+set(CPACK_DEBIAN_PACKAGE_MAINTAINER ${CPACK_PACKAGE_VENDOR})
+
+set(CPACK_DEBIAN_PACKAGE_HOMEPAGE
+    "https://github.com/KhronosGroup/OpenCL-Headers")
+
+# Version number [epoch:]upstream_version[-debian_revision]
+set(LATEST_RELEASE_VERSION "2023.04.17")
+set(CPACK_DEBIAN_PACKAGE_VERSION "${PROJECT_VERSION}~${LATEST_RELEASE_VERSION}")  # upstream_version
+set(CPACK_DEBIAN_PACKAGE_RELEASE "1") # debian_revision (because this is a
+                                      # non-native pkg)
+set(PACKAGE_VERSION_REVISION "${CPACK_DEBIAN_PACKAGE_VERSION}-${CPACK_DEBIAN_PACKAGE_RELEASE}")
+
+set(DEBIAN_PACKAGE_NAME "opencl-c-headers")
```

Comment:
Is this an existing package name or something new?  If this is a new package name I'd suggest removing "c", since I tend to think of "OpenCL C" as being the kernel language.